src/5e-SRD-Traits.json

```diff
@@ -448,13 +448,406 @@
       }
     ],
     "subraces": [],
-    "name": "Draconic Ancestry",
+    "name": "Choose: Draconic Ancestry",
     "desc": [
       "You have draconic ancestry. Choose one type of dragon from the Draconic Ancestry table. Your breath weapon and damage resistance are determined by the dragon type, as shown in the table."
     ],
     "proficiencies": [],
+    "trait_specific": {
+      "subtrait_options": {
+        "choose": 1,
+        "from": [
+          {
+            "index": "draconic-ancestry-black",
+            "name": "Draconic Ancestry (Black)",
+            "url": "/api/traits/draconic-ancestry-black"
+          },
+          {
+            "index": "draconic-ancestry-blue",
+            "name": "Draconic Ancestry (Blue)",
+            "url": "/api/traits/draconic-ancestry-blue"
+          },
+          {
+            "index": "draconic-ancestry-brass",
+            "name": "Draconic Ancestry (Brass)",
+            "url": "/api/traits/draconic-ancestry-brass"
+          },
+          {
+            "index": "draconic-ancestry-bronze",
+            "name": "Draconic Ancestry (Bronze)",
+            "url": "/api/traits/draconic-ancestry-bronze"
+          },
+          {
+            "index": "draconic-ancestry-copper",
+            "name": "Draconic Ancestry (Copper)",
+            "url": "/api/traits/draconic-ancestry-copper"
+          },
+          {
+            "index": "draconic-ancestry-gold",
+            "name": "Draconic Ancestry (Gold)",
+            "url": "/api/traits/draconic-ancestry-gold"
+          },
+          {
+            "index": "draconic-ancestry-green",
+            "name": "Draconic Ancestry (Green)",
+            "url": "/api/traits/draconic-ancestry-green"
+          },
+          {
+            "index": "draconic-ancestry-red",
+            "name": "Draconic Ancestry (Red)",
+            "url": "/api/traits/draconic-ancestry-red"
+          },
+          {
+            "index": "draconic-ancestry-silver",
+            "name": "Draconic Ancestry (Silver)",
+            "url": "/api/traits/draconic-ancestry-silver"
+          },
+          {
+            "index": "draconic-ancestry-white",
+            "name": "Draconic Ancestry (White)",
+            "url": "/api/traits/draconic-ancestry-white"
+          }
+        ],
+        "type": "trait"
+      }
+    },
     "url": "/api/traits/draconic-ancestry"
   },
+  {
+    "index": "draconic-ancestry-black",
+    "races": [
+      {
+        "index": "dragonborn",
+        "name": "Dragonborn",
+        "url": "/api/races/dragonborn"
+      }
+    ],
+    "subraces": [],
+    "name": "Draconic Ancestry (Black)",
+    "desc": [
+      "You have draconic ancestry. Choose one type of dragon from the Draconic Ancestry table. Your breath weapon and damage resistance are determined by the dragon type, as shown in the table."
```

Comment:
Should we be showing the table somewhere? We're also missing breath weapon information?

Reply:
Underneath the subtrait_options under the trait_specific attribute of is a choice which includes all possible draconic ancestry features you can select. This is the only equivalent of the table we need imo.

As for not including breath weapon information underneath trait_specific I did this because it's less concrete. With damage_type and save_ability_scote these API references could actually be passed onto consumers and used to either deal damage or have someone make a saving throw, whereas any breath weapon information would either go unused or have to to individually interpreted. It COULD be added, nontheless.

Reply:
> As for not including breath weapon information underneath trait_specific I did this because it's less concrete. With damage_type and save_ability_scote these API references could actually be passed onto consumers and used to either deal damage or have someone make a saving throw, whereas any breath weapon information would either go unused or have to to individually interpreted. It COULD be added, nontheless.

It *should* be added. Otherwise that data is just completely missing from the API. It could be as simple as:

```json
"breath_weapon": {
  "shape": "cone",
  "size: 15,
}
```

Substituting the values as appropriate, of course. Alternatively, it could be formatted more like the actions and attacks that Monsters have - that would be my preference.

That data *could* be used directly in a highly automated system, but it could also just be used to display information about the trait. Even if it's not being used to automate attacks or whatever, if it's not there, there's nothing in the data representing the Breath Weapon part of the trait.